src/client/MenuWindow.cpp

```diff
@@ -2,186 +2,372 @@
 #include <iostream>
 #include <json/json.h>
 #include <fstream>
+#include <string>
+#include <codecvt>
 
-#define WINDOW_LENGTH 1600
-#define WINDOW_WIDTH 900
+#define REFRESH_TIME 30
 
-#define TITLE_SIZE 200
+#define BUTTON_CREAT 1
+#define BUTTON_USERNAME 2
+#define BUTTON_SERVER 3
+#define BUTTON_PORT 4
+#define BUTTON_LOAD 4
+#define BUTTON_ID 5
+#define BUTTON_PLAYER 5
+#define BUTTON_SEED 6
+#define BUTTON_CONNECT 6
+#define BUTTON_START 7
```

Comment:
Could not these define be in a json file or either in the enum class `HudTextureType` or something like that?